app/controllers/organizations/staff/dashboard_controller.rb

```diff
@@ -1,11 +1,61 @@
 class Organizations::Staff::DashboardController < Organizations::BaseController
+  before_action :context_authorize!, only: %i[index incomplete_tasks overdue_tasks]
+  include Pagy::Backend
   layout "dashboard"
 
   def index
     @user = current_user
     @organization = Current.organization
     @hide_footer = true
+    pet_ids = @organization.pets.pluck(:id)
+    @not_completed_not_overdue_tasks_count = Task.where(pet_id: pet_ids).is_not_completed.not_overdue.count
```

Comment:
So for here, can't we call Task.is_not_completed.not_overdue.count directly? I think this should be querying the tenant scope as this controller inherits from Organizations::BaseController, which includes this module where we set the tenant https://github.com/rubyforgood/pet-rescue/blob/main/app/controllers/concerns/organization_scopable.rb